### PR TITLE
Adds some missing stuff to score ui

### PR DESCRIPTION
**Pull request by MaxIsJoe**
### Changelog:

CL: [New] Adds a UI close button because people forgot that the escape key exists.
CL: [New] Score Rating logic went missing somehow? It's back now.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Messages.Server;
 using Mirror;
 using Systems.Score;
 using TMPro;
+using UnityEditor;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs]: this namespace can't be used in runtime. I don't see you using it anywhere, so you can either delete it or wrap in a ``# if UNITY_EDITOR`` preprocessor

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs]: This namespace shouldn't be even here, deleted.

File: UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs]: what about this one? I don't see any linq in your new code

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs]: yup, removed.

**Comment by corp-0** [comment]
> Your PR has no registered changes in its description!

it fukken does! I didn't modify changes detection, why is it failing now xD

**Comment by corp-0** [comment]
less gooo, it works now. And again I didn't modify change detection so I don't know what happened

**Review by NoooneyDude — DISMISSED** [review]

**Comment by MaxIsJoe** [comment]
I did not mean to do that reee

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**